modules/gapi/include/opencv2/gapi/infer/onnx.hpp

```diff
@@ -122,6 +124,15 @@ template<typename Net> class Params {
         return *this;
     }
 
+    Params<Net>& cfgPostProc(const std::vector<cv::GMatDesc> &outs,
```

Comment:
in most of cases we have `&&` as an input.
Does it make sense to support this case separately

Reply:
Added overloads.